packages/renderer/src/lib/image/ImageColumnName.svelte

```diff
@@ -8,9 +9,15 @@ export let object: ImageInfoUI;
 
 function openDetails(image: ImageInfoUI): void {
   if (image.isManifest) {
-    router.goto(`/manifests/${image.id}/${image.engineId}/${image.base64RepoTag}/summary`);
+    handleNavigation({
+      page: NavigationPage.MANIFEST,
+      parameters: { id: image.id, engineId: image.engineId, tag: image.name + ':' + image.tag },
+    });
   } else {
-    router.goto(`/images/${image.id}/${image.engineId}/${image.base64RepoTag}/summary`);
+    handleNavigation({
+      page: NavigationPage.IMAGE,
+      parameters: { id: image.id, engineId: image.engineId, tag: image.name + ':' + image.tag },
+    });
```

Comment:
I suppose that the `parameters` field is expected to always be the same (now and in the future), we could have the switch in the `page` field only

Reply:
Yeah, I wouldn't expect them to ever differ and that is much simpler - change pushed.